node/node.go

```diff
@@ -1128,3 +1137,8 @@ func (node *Node) CborStore() *hamt.CborIpldStore {
 func (node *Node) ChainReadStore() chain.ReadStore {
 	return node.ChainReader
 }
+
+// DAG is the interface that defines methods to interact with IPLD DAG objects.
+func (node *Node) DAG() dagFormat.DAGService {
```

Comment:
BLOCKING: Avoiding exporting this type of functionality from node is one of the goals of this refactor. This function should be inlined.

Reply:
Removed from the node entirely.